packages/fluentui/e2e/e2eApi.ts

```diff
@@ -28,54 +45,99 @@ export class E2EApi {
   };
 
   public getElement = async (selector: string) => {
-    return await this.page.waitForSelector(selector, { timeout: 10 * 1000 });
+    return await this.page.waitForSelector(selector, { timeout: PUPPETEER_ACTION_TIMEOUT });
   };
 
   public hover = async (selector: string) => {
     return await this.page.hover(selector);
   };
 
-  public getAttributeValue = async (selector: string, attr) => {
-    return this.page.$eval(selector, (el, attribute) => el.getAttribute(attribute), attr);
+  public hasComputedStyle = async (
+    selector: string,
+    property: keyof CSSStyleDeclaration,
+    value: string,
+  ): Promise<void> => {
+    await this.page.waitForFunction(
+      (selector, property, value) => {
+        return window.getComputedStyle(document.querySelector(selector))[property] === value;
+      },
+      { timeout: PUPPETEER_ACTION_TIMEOUT },
+      selector,
+      property,
+      value,
+    );
   };
 
-  public getPropertyValue = async (selector: string, prop) => {
-    return this.page.$eval(selector, (el, prop) => el[prop], prop);
+  public hasPropertyValue = async (selector: string, property: string, value: number | string): Promise<void> => {
+    await this.page.waitForFunction(
+      (selector, property, value) => {
+        return document.querySelector(selector)[property] === value;
+      },
+      { timeout: PUPPETEER_ACTION_TIMEOUT },
+      selector,
+      property,
+      value,
+    );
   };
 
-  public count = async (selector: string) => {
-    return (await this.page.$$(selector)).length;
+  public count = async (selector: string, count: number): Promise<void> => {
```

Comment:
```suggestion
  public expectCount = async (selector: string, count: number): Promise<void> => {
```
or `hasCount` or anything which implies this is an assertion.

The same applies for `hidden` and `textOf`.